addons/web/static/src/js/libs/pdfjs.js

```diff
@@ -1,5 +1,38 @@
+/** @odoo-module **/
+
+import config from 'web.config';
+
+export function hidePDFJSButtons(rootElement) {
```

Comment:
It would be nice to have a little docstring explaining what it does and especially why this is necessary. It's better if the code contains the info instead of relying on blaming the commit.